docs/tutorials/index.rst

```diff
@@ -21,88 +21,125 @@ handbook`_ and the `Astropy Hands-On Tutorial`_.
-   ../notebooks/models.ipynb
-   ../notebooks/catalog.ipynb
 
-**Getting started**
+The following tutorials expose common tasks.
 
-The following tutorials show the same 3D cube analysis of the Crab with the high level
-interface and with the lower level API:
-
-- `First analysis <../notebooks/analysis_1.html>`__ | *analysis_1.ipynb*
-- `Second analysis <../notebooks/analysis_2.html>`__ | *analysis_2.ipynb*
-
-**What data can I analyse?**
+*Accessing and exploring DL3 data*
 
+- `Overview <../notebooks/overview.html>`__  | *overview.ipynb*
 - `CTA with Gammapy <../notebooks/cta.html>`__ | *cta.ipynb*
 - `H.E.S.S. with Gammapy <../notebooks/hess.html>`__ |  *hess.ipynb*
-- `Fermi-LAT with Gammapy <../notebooks/fermi_lat.html>`__ | *fermi_lat.ipynb*
 
-**Analyses**
-
-*3-dim sky cube analysis*
+*1D spectral analysis*
 
-- `CTA data analysis with Gammapy <../notebooks/cta_data_analysis.html>`__ | *cta_data_analysis.ipynb*
-- `3D analysis <../notebooks/analysis_3d.html>`__ | *analysis_3d.ipynb*
-- `Multi instrument joint 3D and 1D analysis <../notebooks/analysis_mwl.html>`__ | *analysis_mwl.ipynb*
+- `Spectral analysis  <../notebooks/spectrum_analysis.html>`__ | *spectrum_analysis.ipynb*
+- `Flux point fitting <../notebooks/sed_fitting.html>`__ | *sed_fitting.ipynb*
 
 *2-dim sky image analysis*
 
 - `Ring background map creation <../notebooks/ring_background.html>`__ | *ring_background.ipynb*
-- `Source detection <../notebooks/detect.html>`__ | *detect.ipynb*
 - `2D map fitting <../notebooks/modeling_2D.html>`__ | *modeling_2D.ipynb*
 
-*1-dim spectral analysis*
+*3-dim sky cube analysis*
 
-- `Spectral analysis  <../notebooks/spectrum_analysis.html>`__ | *spectrum_analysis.ipynb*
-- `Flux point fitting <../notebooks/sed_fitting.html>`__ | *sed_fitting.ipynb*
+- `CTA data analysis with Gammapy <../notebooks/cta_data_analysis.html>`__ | *cta_data_analysis.ipynb*
+- `3D analysis <../notebooks/analysis_3d.html>`__ | *analysis_3d.ipynb*
 
 *Time-dependent analysis*
 
 - `Light curves <../notebooks/light_curve.html>`__ | *light_curve.ipynb*
 - `Light curves for flares <../notebooks/light_curve_flare.html>`__ | *light_curve_flare.ipynb*
 
-**Simulations, Sensitivity, Observability**
+*Simulations, Sensitivity, Observability*
 
-- `3D map simulation <../notebooks/simulate_3d.html>`__ | *simulate_3d.ipynb*
 - `1D spectrum simulation <../notebooks/spectrum_simulation.html>`__ | *spectrum_simulation.ipynb*
+- `3D map simulation <../notebooks/simulate_3d.html>`__ | *simulate_3d.ipynb*
+
+Advanced tutorials
+------------------
+
+.. toctree::
+   :hidden:
+
+   ../notebooks/analysis_mwl.ipynb
+   ../notebooks/detect.ipynb
+   ../notebooks/cta_sensitivity.ipynb
+   ../notebooks/spectrum_simulation.ipynb
+   ../notebooks/modeling_2D.ipynb
+   ../notebooks/ring_background.ipynb
+   ../notebooks/overview.ipynb
+   ../notebooks/maps.ipynb
+   ../notebooks/modeling.ipynb
+   ../notebooks/models.ipynb
+   ../notebooks/catalog.ipynb
+
+*Image analysis*
```

Comment:
Is this separate section needed or this can also go under 2-d image analysis?

Reply:
Added specific info on detection